src/modules/projects/ExternalSubmissionsViewModal.tsx

```diff
@@ -0,0 +1,165 @@
+import {
```

Comment:
can we move this up into the "components" dir?

Reply:
Yes, sorry!